src/cls/IPM/Utils/Module.cls

```diff
@@ -389,8 +389,7 @@ ClassMethod LoadModuleFromArchive(pModuleName As %String, pModuleVersion As %Str
 	Try {
 		Set tVerbose = $Get(pParams("Verbose"))
 		
-		Set tTargetDirectory = ##class(%File).ManagerDirectory()_".modules\"_$Namespace_"\"_pModuleName_"\"_pModuleVersion_"\"
-		Set tTargetDirectory = ##class(%File).NormalizeDirectory(tTargetDirectory)
+		Set tTargetDirectory = ##class(%SYS.System).TempDirectory()
```

Comment:
Really want a subdirectory of this.

Reply:
Subdirectory can be the module name and should check if the directory exists first and delete it if so. Also, probably good to delete the directory at the end of the run